data/custom.xml

```diff
@@ -55,7 +55,7 @@
 
     <magic priority="50">
       <match value="PK\003\004" type="string" offset="0">
-        <match value="[Content_Types].xml" type="string" offset="30">
+        <match value="[Content_Types].xml" type="string" offset="30:65536">
```

Comment:
At least for a blank xlsx file 4096 would be sufficient. I used a common range for the three different format though, so as to make these numbers less cryptic.